crates/swc_ecma_parser/tests/tsc/autoAccessor1.ts

```diff
@@ -6,4 +6,5 @@ class C1 {
     accessor b = 1;
     static accessor c: any;
     static accessor d = 2;
+    accessor e!: string;
```

Comment:
Please revert this. `tests/tsc` of the parser is copied from the conformance test suite of TypeScript

Reply:
Done.